### PR TITLE
Simplify FileManager API:

### PR DESCRIPTION
**Pull request by KevinFairise2**
What does this PR do?
---------------------

Simplify file manager API by removing the need to set a resource name

Which scenarios this will impact?
-------------------

Motivation
----------

Have a simpler API
Additional Notes
----------------

### CODE REVIEW

File: components/command/filemanager.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func (fm *FileManager) CopyAbsoluteFolder(absoluteFolder string, remoteFolder st
 	baseFolder := filepath.Base(absoluteFolder)
 	rootWithoutBase := absoluteFolder[:len(absoluteFolder)-len(baseFolder)]
 	// Use remoteFolder as `absoluteFolder` may be a random file path that is different for each run.
```
Comment (pducolin) [inline components/command/filemanager.go:81]: We can remove this comment, underlying we keep the same behaviour 
```suggestion
```

**Review by pducolin — APPROVED** [review]
Tested on a temporary scenario, it works fine for both temp and home directories !

**Comment by vboulineau** [comment]
This removes the ability to create folder from `Output`, that's why the simplification was not made in the past.

**Review by ogaca-dd — APPROVED** [review]

**Comment by pducolin** [comment]
> This removes the ability to create folder from Output, that's why the simplification was not made in the past.

After discussion, we'll add a `CreateFolderWithPulumiString` for such a use case - note so far it is not used.

**Pull request merged**